test/hip-583/HIP583.js

```diff
@@ -537,13 +513,15 @@ describe('HIP583 Test Suite - Contract Transfer TX', function () {
 
     before(async function () {
       hollowWallet = ethers.Wallet.createRandom().connect(ethers.provider)
+
       await (
         await contractTransferTx.transferTo(
           hollowWallet.address,
-          amount / utils.tinybarToWeibarCoef,
+          (amount * 20) / utils.tinybarToWeibarCoef,
```

Comment:
Q: why the factor of 20 here and below?

Reply:
That value was determined by @quiet-node  to allow the test to pass at the time.  I was able to reduce it to 14.

Reply:
Unlike the prior cases this is multiplying the current `amount` by 14. Why multiple vs state an amount?

Reply:
Updated to a stated amount.